modules/types/src/channelProvider.ts

```diff
@@ -43,39 +84,28 @@ export const ChannelProviderRpcMethods = {
 };
 export type ChannelProviderRpcMethod = ConnextRpcMethod | CFCoreTypes.RpcMethodName;
 
+export type JsonRpcRequest = {
+  id: number;
+  jsonrpc: "2.0";
+  method: string;
+  params: any;
+};
```

Comment:
is this exported from cf or no?

Reply:
vscode wasn't returning anything but I can search again

Reply:
Actually it doesn't because it uses a different schema
```typescript
export type Rpc = {
  methodName: string;
  parameters: RpcParameters;
  id?: number;
};
```